app/Domains/Contact/ManageContact/Dav/ImportContact.php

```diff
@@ -114,22 +114,17 @@ private function getExistingContact(VCard $vcard): ?Contact
      */
     private function getContactData(?Contact $contact): array
```

Comment:
Why would we call this function without a contact?

Reply:
It happens when we import a new vcard for the 1st time